### PR TITLE
filter MediaRelationship by role

### PR DESCRIPTION
**Pull request by matthewdias**

### CODE REVIEW

File: app/resources/media_relationship_resource.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class MediaRelationshipResource < BaseResource
   attribute :role
+
```
[the remark this answers is not in the document]
Comment (matthewdias) [reply app/resources/media_relationship_resource.rb]: lol that's what i get for using the github editor

**Comment by toyhammered** [comment]
best PR ever lol.

**Comment by NuckChorris** [comment]
`role` is actually an enum, I think we need to do this differently to allow filtering by string-form... there's other models in the codebase that do this, I think `GroupMemberResource` might have something you could reference?

**Comment by matthewdias** [comment]
i'll check it out 👍

File: app/resources/media_relationship_resource.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (wopian) [reply app/resources/media_relationship_resource.rb]: `server` needs an [`.editorconfig`](https://github.com/hummingbird-me/hummingbird-client/blob/the-future/.editorconfig) like `client` @NuckChorris!

**Pull request merged**